### DISCUSSION TITLE
Excessive number of guests head to new rides in large old parks resulting in overcrowding

### DISCUSSION THREAD

**Discussion by CSEverett** [OpenRCT2/OpenRCT2#15828] [Feature requests]
### Operating System

Windows 7, 64 bit

### OpenRCT2 build

OpenRCT2, v0.3.4.1 (5087e77)

### Describe the issue

I'm playing old, mature parks that have been in existence for 20+ years of in-game time, and have around 7000 guests with not very many rides built in the last 5 years or so. Most of the major rides are a decade or more old. Although just about everything gets refurbished, which 'should' reset ride age, every few years. But now every time I open a new ride, hundreds (as many as 500) of guests immediately head for the new ride. I've had 110 people heading for a Twist. Which inevitably overloads the ride and the footpath system, resulting in overcrowding and hundreds of guests leaving the park. And the other part of the problem is that using the "renew all rides" cheat either before, after or both opening the ride doesn't prevent it. And I'm using at least "double tracked" (two tiles with a fence between) footpaths in these cases, and I'm even having overcrowding on "four tracked" footpaths. Whatever the fix is, the problem means opening any new ride in these types of parks usually requires a carefully planned replacement of multiple rides around the park. Probably should either require a change in guest behavior, or improvements to what the "renew all rides" cheat.

Right now it results in major difficulty in adding any new rides to large, old parks. I've heard others have the same problem. Hopefully this will be fixed in the 3.5 release?

### Steps to reproduce

1. Have a park 20+ years old with around 7000 guests.
2. Have most of the big rides be several years to decades old, with refurbishing them every few years
3. Open a new ride, particularly one that can't handle 3500+ guests per hour.

**Comment by duncanspumpkin** [comment]
This isn't really a bug. This is just about late game balance and should be a discussion not an issue.

**Comment by CSEverett** [comment]
Some screenshots, showing guest density/actions/thoughts:
![Screen Shot 10-26-21 at 04 23 PM 001](https://user-images.githubusercontent.com/89952921/139438280-8dec2743-768e-49fc-909c-6e3a75d3892f.PNG)
![Screen Shot 10-26-21 at 04 23 PM](https://user-images.githubusercontent.com/89952921/139438346-86dcc4b7-d7c3-43c2-8898-4c292609ea53.PNG)
![Screen Shot 10-26-21 at 04 22 PM](https://user-images.githubusercontent.com/89952921/139438418-81e02f92-fd4c-4fff-a514-04ac1c1c20e0.PNG)
:

**Comment by CSEverett** [comment]
> This isn't really a bug. This is just about late game balance and should be a discussion not an issue.

Sooo, I should repost this, where...? Don't really know how to use this site.

**Comment by J-PIE-314** [comment]
When this was moved to discussions it became a duplicate of #15825.
